homeworks/HW3-final/Bank.py

```diff
@@ -0,0 +1,173 @@
+from enum import Enum
```

Comment:
-2 point: Savings account hasnt been created yet but getbalance() does not raise Exception,
-1 point: Savings account hasnt been created yet but withdraw does not raise an Exception.,
-1 point: Savings account hasnt been created yet but deposit does not raise an Exception.,
-6 points: ATMSession did not update CHECKING account properly.